website/data/plugins-manifest.json

```
@@ -230,6 +230,7 @@
     "repo": "outscale/packer-plugin-outscale",
     "version": "latest",
     "pluginTier": "verified"
```

Since the `pluginTier` line does not end with a `,` this is invalid JSON

```suggestion
    "pluginTier": "verified",
```